back/libs/logger-legacy/src/services/pino.service.ts

```diff
@@ -1,53 +0,0 @@
-import pino, { DestinationStream, Logger } from 'pino';
-
-import { Injectable, ShutdownSignal } from '@nestjs/common';
-
-import { ConfigService } from '@fc/config';
-
-import { LoggerConfig as LoggerLegacyConfig } from '../dto';
-import { LoggerLevelCodes } from '../enum';
-import { pinoLevelsMap } from '../log-maps.map';
-
-/**
- * Formatter for pino library
- * @see https://github.com/pinojs/pino/blob/master/docs/api.md#formatters-object
- */
-export function formatLevel(label: string, _number: number): { level: string } {
```

Comment:
question: I don't really understand that part? why we needed a "custom" pino service before?

What I understand here, it's that we needed a custom stream build with `getDestinationForLog`?